### PR TITLE
net.minecraft.network stuff

### PR DESCRIPTION
**Pull request by NoahvdAa**

### CODE REVIEW

File: mappings/net/minecraft/network/packet/s2c/play/PlayerActionResponseS2CPacket.mapping [hunk 1]
```diff
@@ -12,5 +17,9 @@ CLASS net/minecraft/unmapped/C_zpyakjnm net/minecraft/network/packet/s2c/play/Pl
 		ARG 5 reason
 	METHOD <init> (Lnet/minecraft/unmapped/C_idfydwco;)V
 		ARG 1 buf
-	METHOD m_vuxhptki getAction ()Lnet/minecraft/unmapped/C_cbvghftd$C_dowtgrks;
-	METHOD m_yzmjggck isApproved ()Z
```
Comment (NoahvdAa) [inline mappings/net/minecraft/network/packet/s2c/play/PlayerActionResponseS2CPacket.mapping]: This class is a record, so these were changed to match the variable names

**Review by triphora — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]
Looks good to me 👍🏼

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 2]
```diff
@@ -707,5 +846,257 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
...
+		ARG 3 charset
+	METHOD setDouble setDouble (ID)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setFloat setFloat (IF)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setIndex setIndex (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 oldIndex
+		ARG 2 newIndex
```
Comment (IotaBread) [inline mappings/net/minecraft/network/PacketByteBuf.mapping]: From [Conventions.md: Getters, setters, withers, and creators](https://github.com/QuiltMC/quilt-mappings/blob/1.18.2/CONVENTIONS.md#getters-setters-withers-and-creators)
> Name the parameter the same as the property (`setColor(color)`, not `setColor(newColor)`).

```suggestion
		ARG 2 index
```

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 3]
```diff
@@ -111,8 +111,147 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
...
+		ARG 1 index
+		ARG 2 gatheringByteChannel
+		ARG 3 length
+	METHOD getBytes getBytes (I[B)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 destination
+	METHOD getBytes getBytes (I[BII)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 destination
+		ARG 3 destinationindex
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/PacketByteBuf.mapping]:
```suggestion
		ARG 3 destinationIndex
```

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 4]
```diff
@@ -707,5 +846,257 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
...
+		ARG 2 bytes
+		ARG 3 sourceIndex
+		ARG 4 length
+	METHOD setChar setChar (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 value
+	METHOD setCharSequence setCharSequence (ILjava/lang/CharSequence;Ljava/nio/charset/Charset;)I
+		ARG 1 index
+		ARG 2 charSequence
+		ARG 3 charset
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/PacketByteBuf.mapping:997]:
```suggestion
		ARG 3 charSet
```
?

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 5]
```diff
@@ -707,5 +846,257 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
...
+	METHOD skipBytes skipBytes (I)Lio/netty/buffer/ByteBuf;
+		ARG 1 length
+	METHOD slice slice ()Lio/netty/buffer/ByteBuf;
+	METHOD slice slice (II)Lio/netty/buffer/ByteBuf;
+		ARG 1 index
+		ARG 2 length
+	METHOD toString toString (IILjava/nio/charset/Charset;)Ljava/lang/String;
+		ARG 1 index
+		ARG 2 length
+		ARG 3 charset
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/PacketByteBuf.mapping:1043]:
```suggestion
		ARG 3 charSet
```
?

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 6]
```diff
@@ -707,5 +846,257 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
...
+	METHOD toString toString (Ljava/nio/charset/Charset;)Ljava/lang/String;
+		ARG 1 charset
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/PacketByteBuf.mapping:1045]:
```suggestion
		ARG 1 charSet
```
?

File: mappings/net/minecraft/network/PacketByteBuf.mapping [hunk 7]
```diff
@@ -707,5 +846,257 @@ CLASS net/minecraft/unmapped/C_idfydwco net/minecraft/network/PacketByteBuf
...
+		ARG 1 bytes
+	METHOD writeBytes writeBytes ([BII)Lio/netty/buffer/ByteBuf;
+		ARG 1 bytes
+		ARG 2 sourceIndex
+		ARG 3 length
+	METHOD writeChar writeChar (I)Lio/netty/buffer/ByteBuf;
+		ARG 1 value
+	METHOD writeCharSequence writeCharSequence (Ljava/lang/CharSequence;Ljava/nio/charset/Charset;)I
+		ARG 1 charSequence
+		ARG 2 charset
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/network/PacketByteBuf.mapping:1086]:
```suggestion
		ARG 2 charSet
```
?

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/network/PacketByteBuf.mapping (hunk 4 above)
Comment (NoahvdAa) [reply mappings/net/minecraft/network/PacketByteBuf.mapping:997]: Class name is `Charset`, adding camelcase here is unnecessary imo

**Comment by NoahvdAa** [comment]
Updated to include 22w11a stuff, excluding changes to the command tree packet (don't want to conflict with whoevers doing that)

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/network/packet/s2c/AcknowledgeBlockChangeS2CPacket.mapping [hunk 8]
```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/unmapped/C_ohnbgqmp net/minecraft/network/packet/s2c/AcknowledgeBlockChangeS2CPacket
+	FIELD f_ncmksjrw sequence I
+	METHOD <init> (I)V
+		ARG 1 sequence
+	METHOD <init> (Lnet/minecraft/unmapped/C_idfydwco;)V
+		ARG 1 buf
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
+	METHOD m_rbfsyvrn getSequence ()I
```
Comment (IotaBread) [inline mappings/net/minecraft/network/packet/s2c/AcknowledgeBlockChangeS2CPacket.mapping]:
```suggestion
	METHOD m_rbfsyvrn sequence ()I
```
I'm guessing this is a record

**Pull request merged**